activiti-cloud-services-runtime-bundle/activiti-cloud-services-core/src/main/java/org/activiti/cloud/services/core/SecurityPoliciesApplicationService.java

```diff
@@ -83,6 +90,11 @@ public ProcessInstanceQuery restrictProcessInstQuery(ProcessInstanceQuery query,
         Set<String> keys = definitionKeysAllowedForRBPolicy(securityPolicy);
 
         if(keys != null && !keys.isEmpty()){
+
+            if(keys.contains(securityPoliciesService.getWildcard())){
```

Comment:
The content of `restrictProcessDefQuery` and `restrictProcessInstDef` are quite similar. Actually, the content is identical, but they are calling a method with the same name declared in different interfaces (ProcesDefinitionQuery.processDefinitionKeys(...) x ProcessInstanceQuery.processDefinitionKeys(...)). I'm wondering if we can reduce the amount of duplication. Maybe if we use local variables instead of early returns.